ui/shared/components/form/XHRUploaderWithEvents.jsx

```diff
@@ -81,9 +81,8 @@ class XHRUploaderWithEvents extends XHRUploader {
         }
       }
       xhr.upload.onprogress = (e) => {
-        if (e.lengthComputable) {
-          progressCallback(e.loaded / e.total * 100) // eslint-disable-line no-mixed-operators
-        }
+        const progress = e.lengthComputable ? (e.loaded / e.total * 100) : 50 // eslint-disable-line no-mixed-operators
+        progressCallback(progress === 100 ? 80 : progress)
```

Comment:
why do you send 80 back to `progressCallback` if `progress` is 100? 

how about making 80 an all-caps constant since it's a specific value that has some meaning?

Reply:
because it is incorrectly reporting progress as 100% in many cases, and we have an explicit `progressCallback(100)` on the actual upload finish event, so having it hover at 80% until that gets triggered means the user never is shown that its finished when it isn't. I chose 80 because thats what looks like a lot has loaded but is still very obvious that its not done, much higher than that and it looks done at. glance